TagsterWebAPI/src/Tagster.Application/Commands/Auth/External/ExternalAuth.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Tagster.Auth.Dtos;
+using Tagster.CQRS.Commands;
+
+namespace Tagster.Application.Commands.Auth.External;
+
+public class ExternalAuth : ICommand<AuthDto>
+{
+    public string Email { get; }
+    public ExternalAuth(string email)
+    {
+        ArgumentNullException.ThrowIfNull(email);
```

Comment:
Create new exception and throw it instead of using system exception. Use one of exception abstraction AppException, DomainException